profiles/thelabuzov.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/87143407/211593786-8cd5f52c-d550-4b46-af6a-9f56fab5f603.png",
+  "issueId": 880,
+  "name": "Dmitry Labuzov",
+  "username": "thelabuzov"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```